docs/scripts_md/updateMRI_Upload.md

```diff
@@ -0,0 +1,52 @@
+# NAME
+
+updateMRI\_Upload.pl - updates database table mri\_upload according to an entry in table
+   tarchive
+
+# SYNOPSIS
+
+updateMRI\_Upload.pl \[options\] -profile prod -tarchivePath tarchivePath -source\_location source\_location
+
+- **-profile prof** : (mandatory) path (absolute or relative to the current directory) of the 
+    profile file
+- **-tarchivePath tarchivePath** : (mandatory) absolute path to the tarchive file
+- **-source\_location source\_location** : (mandatory) value to set column 
+    DecompressedLocation to for the newly created record in table mri\_upload (see below)
+- **-globLocation** : Loosen the validity check of the tarchive allowing for the 
+     possibility that the tarchive was moved to a different directory.
+- **-verbose** : Be verbose
```

Comment:
I like how you bolded the options; I wish we did that ourselves as well; what do you think @cmadjar ?

Reply:
Yep. Never too late. We could make sure this is harmonized once all the scripts are documented? Probably easier to do it once than change each PR ;).